update_spreadsheet.py

```diff
@@ -18,15 +18,14 @@
 class UpdateSpreadsheet(object):
     def __init__(self, env):
         self.env = env
-        self.jst = self.set_jst_now()
+        self.jst = None
 
-    @staticmethod
-    def set_jst_now():
-        tokyo_timezone = pytz.timezone('Asia/Tokyo')
-        utc = datetime.now()
-        jst = tokyo_timezone.fromutc(utc)
+    def set_jst_now(self, jst=None):
+        if jst is None:
+            tokyo_timezone = pytz.timezone('Asia/Tokyo')
+            utc = datetime.now()
+            self.jst = tokyo_timezone.fromutc(utc)
 
```

Comment:
jst を引数にしたのは、もしかして: 
```suggestion
        else:
            self.jst = jst
```
という意向かな？

Reply:
そうです！
テストしやすいようにと思って。
else付け足すの忘れてました🧟‍♂️